entities/order.rb

```diff
@@ -16,6 +16,6 @@ def initialize(book:, reader:, date:)
   end
 
   def validate(*date)
```

Comment:
why do you need splat here?
`def validate(date)`?

Reply:
done